### PR TITLE
Mdoc 5.9.2.1 release

### PR DESCRIPTION
**Pull request by huangmin-ms**
## Summary
Fix:
1.	[Bug 715817: mdoc generates Attribute node for 1 class but not the other - Boards (visualstudio.com)](https://ceapex.visualstudio.com/Engineering/_workitems/edit/715817)
2.	[Bug 755541: [Dotnet] Doc comments are imported even when option is unchecked - Boards (visualstudio.com)](https://ceapex.visualstudio.com/Engineering/_workitems/edit/755541)
3. Fix member order diffs by sorting the member name alphabetically.

### CODE REVIEW

File: xml/Microsoft.Extensions.Logging.AzureAppServices.Internal/BlobAppendReferenceWrapper.xml [hunk 1]
```diff
@@ -27,32 +27,24 @@
...
       <AssemblyInfo>
         <AssemblyName>Microsoft.Extensions.Logging.AzureAppServices</AssemblyName>
         <AssemblyVersion>1.0.0.0</AssemblyVersion>
       </AssemblyInfo>
       <Parameters>
-        <Parameter Name="cloudAppendBlob" Type="Microsoft.Azure.Storage.Blob.CloudAppendBlob" Index="0" FrameworkAlternate="dotnet-plat-ext-1.1" />
+        <Parameter Name="cloudAppendBlob" Type="Microsoft.WindowsAzure.Storage.Blob.CloudAppendBlob" Index="0" FrameworkAlternate="dotnet-plat-ext-1.1" />
       </Parameters>
       <Docs>
-        <param name="cloudAppendBlob">The <see cref="T:Microsoft.Azure.Storage.Blob.CloudAppendBlob" /> instance to wrap.</param>
```
Comment (gewarren) [inline xml/Microsoft.Extensions.Logging.AzureAppServices.Internal/BlobAppendReferenceWrapper.xml:45]: There's some data loss here.

**Review by gewarren — APPROVED** [review]
Looks good to me other than the one area of data loss.

File: xml/Microsoft.Extensions.Logging.AzureAppServices.Internal/BlobAppendReferenceWrapper.xml (hunk 1 above)
Comment (huangmin-ms) [reply xml/Microsoft.Extensions.Logging.AzureAppServices.Internal/BlobAppendReferenceWrapper.xml:45]: @gewarren  
public BlobAppendReferenceWrapper (Microsoft.**WindowsAzure**.Storage.Blob.CloudAppendBlob cloudAppendBlob); 
is a new method and it matches the source code.
image
The old method is deleted with its documentation.

Comment (gewarren) [reply xml/Microsoft.Extensions.Logging.AzureAppServices.Internal/BlobAppendReferenceWrapper.xml:45]: Okay, I had changed the namespace in https://github.com/dotnet/dotnet-api-docs/pull/8746 to fix the warnings here: https://github.com/dotnet/dotnet-api-docs/pull/8745#issuecomment-1366967885. But it looks like those warnings are gone now anyway. I'll make a commit here to retain the documentation comments.

File: xml/Microsoft.Extensions.Logging.AzureAppServices.Internal/BlobAppendReferenceWrapper.xml [hunk 2]
```diff
@@ -27,32 +27,24 @@
...
-        <remarks>
-          <format type="text/markdown"><![CDATA[
-
-## Remarks
-
-This API supports the .NET infrastructure and is not intended to be used directly from your code.
-
-]]></format>
-        </remarks>
+        <param name="cloudAppendBlob">To be added.</param>
```
Comment (gewarren) [inline xml/Microsoft.Extensions.Logging.AzureAppServices.Internal/BlobAppendReferenceWrapper.xml:45]:
```suggestion
        <param name="cloudAppendBlob">The blob instance to wrap.</param>
```

File: xml/Microsoft.Extensions.Logging.AzureAppServices.Internal/BlobAppendReferenceWrapper.xml [hunk 3]
```diff
@@ -27,32 +27,24 @@
...
+        <summary>To be added.</summary>
```
Comment (gewarren) [inline xml/Microsoft.Extensions.Logging.AzureAppServices.Internal/BlobAppendReferenceWrapper.xml:46]:
```suggestion
        <summary>Creates a new instance of <see cref="T:Microsoft.Extensions.Logging.AzureAppServices.Internal.BlobAppendReferenceWrapper" />.</summary>
```

File: xml/Microsoft.Extensions.Logging.AzureAppServices.Internal/BlobAppendReferenceWrapper.xml [hunk 4]
```diff
@@ -27,32 +27,24 @@
...
+        <remarks>To be added.</remarks>
```
Comment (gewarren) [inline xml/Microsoft.Extensions.Logging.AzureAppServices.Internal/BlobAppendReferenceWrapper.xml:47]:
```suggestion
        <remarks>This API supports the .NET infrastructure and is not intended to be used directly from your code.</remarks>
```

**Pull request merged**